runtime-modules/forum/src/lib.rs

```diff
@@ -1153,6 +952,9 @@ impl<T: Trait> Module<T> {
         Self::ensure_can_mutate_in_path_leaf(&category_tree_path)
     }
 
+    // Clippy linter warning
+    #[allow(clippy::ptr_arg)] // disable it because of possible frontend API break
```

Comment:
The problem here is not a method but a result type:
` &CategoryTreePath<T::BlockNumber, T::Moment, T::AccountId>,`

The linter suggested to replace it with an array pointer because CategoryTreePath is a Vec alias, and this, in turn, can break the API.